core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -144,6 +144,11 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
       ThreadUtils.newDaemonSingleThreadScheduledExecutor("cleanup-decommission-execs")
     }
 
+  override def getDriverLogUrls: Option[Map[String, String]] = {
+    val logUrlHandler = new ExecutorLogUrlHandler(conf.get(UI.CUSTOM_DRIVER_LOG_URL))
```

Comment:
Although I understand that you are reusing this, it looks strange to `ExecutorLogUrlHandler` for `getDriverLogUrls`. Do you think we can generalize `ExecutorLogUrlHandler` name?

Reply:
I have moved `spark.executor.ExecutorLogUrlHandler` to `spark.LogUrlHandler`.